starter-code/public/index.html

```diff
@@ -38,8 +38,8 @@
         <div class="icon-menu"></div>
         <ul>
           <!-- TODO: Refactor the href attributes in the Home and About anchor tags to navigate users to our new client-side routes -->
-          <li class="tab" data-content="articles"><a href="#" class="icon-home"> Home</a></li>
-          <li class="tab" data-content="about"><a href="#" class="icon-address-book"> About</a></li>
+          <li class="tab" data-content="articles"><a href="./" class="icon-home"> Home</a></li>
+          <li class="tab" data-content="about"><a href="./about" class="icon-address-book"> About</a></li>
```

Comment:
Note, just a '/' is fine. Think about how URLs look, you usually only have '.' before 'com', 'org', etc. This is what will project onto the URL string